README.md

```diff
@@ -11,9 +11,10 @@ ready-to-deploy images.
 As a starting point please see [example docker-compose configuration](https://github.com/pimcore/skeleton/blob/HEAD/docker-compose.yaml).
 
 ## Image flavors
-We're providing 4 different image flavors: 
+We're providing different image flavors: 
 - PHP image for FPM and CLI with minimum requirements (e.g. `php8.1-min-latest` or `php8.2-min-latest`)
-- PHP image for FPM and CLI incl. all optional dependencies (e.g. `php8.1-latest` or `php8.2-latest`)
+- PHP image for FPM and CLI incl. all optional dependencies (e.g. `php8.1-latest` or `php8.2-latest`) - **recommended**
+- PHP image for FPM and CLI incl. all optional dependencies and additional extensions and software (e.g. `php8.1-max-latest` or `php8.2-max-latest`)
```

Comment:
There are no `php8.1` images in `v2` (the same for `php8.1-min-latest`, which does not exist).